packages/cubejs-schema-compiler/test/PreAggregationsTest.js

```diff
@@ -300,6 +317,67 @@ describe('PreAggregations', function test() {
     });
   });
 
+  it('leaf measure pre-aggregation with measure filter', () => {
+    return compiler.compile().then(() => {
+      const query = new PostgresQuery({ joinGraph, cubeEvaluator, compiler }, {
+        measures: [
+          'visitors.count',
+          'visitors.google_count',
+        ],
+        timeDimensions: [{
+          dimension: 'visitors.createdAt',
+          granularity: 'date',
+          dateRange: ['2017-01-01', '2017-01-30']
+        }],
+        dimensions: ['visitors.source'],
```

Comment:
add test case that doesn't use this dimension. should still use this preaggregation